src/pages/Login/Logo/Logo.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Logo = ({ src ,alt, style1 , style2}) => {
+  return (
+    <h1 className={`${style1}`}>
+      <img src={src} alt={alt} className={`${style2}`} />
+    </h1>
+  )
+}
+
+export default Logo
```

Comment:
This component is identical to ImageComponent.
The only difference is style2. You could have bind it conditionally instead of creating a new component.